fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -176,6 +176,9 @@
 
     public static final String LDPCV_BINARY_TIME_MAP = "fedora:binaryTimemap";
 
+    @VisibleForTesting
```

Comment:
This annotation can be removed. 
`CONTAINER_WEBAC_ACL` is also used in the non-test context of `HttpResourceConverter`.